### PR TITLE
[EDA] Field Encryption Is Failing on ACCT_IndividualsAccount_TEST

### LINKED ISSUE #1105: [EDA] Field Encryption Is Failing on ACCT_IndividualsAccount_TEST
**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**

* **Do you want to request a *feature* or report a *bug*?**
Bug
* **What is the current behavior?**
Customers who have Contact Name encrypted have not been able to get push upgrade to 1.92.
* **What is the expected behavior?**
Customers who have Contact Name encrypted should be able to get push upgrade to 1.92. 
* **What is the motivation / use case for changing the behavior?**
Customers need to be able to receive the newest feature and bug updates from 1.92. 
* **Acceptance Criteria**

  - **Given**: A customer has enabled encryption on Contact "Name" field
  - **When**: When Salesforce tries to upgrade customer org to 1.92
  - **Then**: the customer org should be eligible to the upgrade without issues

### PR DESCRIPTION
**Pull request by kdy618**
# Critical Changes

# Changes
We've made an update that enables customers who encrypted the Name field on the Contact object to successfully install EDA release 1.92.

# Issues Closed
Closes #1105

# New Metadata

# Deleted Metadata

# Testing Notes
Pre-requisite: 
- Enable encryption
- Set "Name" encrypted on Contact

Scenario 1: Bulk test: Update x amount of Contacts from different Accounts and mark the "Deceased" or "Exclude From Household Naming" checkboxes true.
Steps:

Create multiple Contacts and Accounts
Ensure you have multiple Contacts on the same Account
Ensure you have multiple Accounts
Update 1 Contact from each Account by marking either "Deceased" or "Exclude From Household Naming".
All Account names should be updated with just the Contacts left on the Account that are not marked Deceased or Excluded from Household Naming.
Scenario 2: Bulk test: Update x amount of Contacts from different Accounts and mark some of the Contacts "Deceased" or "Exclude From Household Naming" checkboxes true, and some Contacts update their FirstName or LastName but do not mark them as "Deceased" or "Exclude From Household Naming"
Steps:

Create multiple Contacts and Accounts
Ensure you have multiple Contacts on the same Account
Ensure you have multiple Accounts
Update 1 Contact from each Account by marking either "Deceased" or "Exclude From Household Naming" or updating their FirstName or LastName.
All Account names should be updated with just the Contacts left on the Account that are not marked "Deceased" or "Excluded from Household Naming" and the Contacts that were updated but nor marked "Deceased" or "Excluded from Household Naming" should still reflect on the Account name.
Scenario 3: Non bulk: Update 1 Contact from an Account and marked it "Deceased" or "Exclude From Household Naming".
Steps:

Create an Account with multiple Contacts
Update 1 Contact and marked "Deceased" or "Exclude From Household Naming"
Account name should be updated with just the Contacts left on the Account that are not marked "Deceased" or "Exclude From Household Naming"

### CODE REVIEW

File: src/classes/ACCT_IndividualAccounts_TEST.cls [hunk 1]
```diff
@@ -1745,13 +1745,13 @@ private class ACCT_IndividualAccounts_TEST {
                 String FirstName3= 'ConThirdName ' + i;
                 String LastName = 'ConLastName ' + i;
 
-            	System.assertEquals(LastName + ' (' + FirstName1 + ' and ' +
-                					FirstName3 + ') ' + ' Household', returnAccounts[i].Name);
+                System.assertEquals(LastName + ' (' + FirstName1 + ' and ' +
```
Comment (lifewithryan) [inline src/classes/ACCT_IndividualAccounts_TEST.cls]: @kdy we seem to be relying on the order of the results here. Anything else we can try?

File: src/classes/ACCT_IndividualAccounts_TEST.cls [hunk 2]
```diff
@@ -1767,7 +1767,7 @@ private class ACCT_IndividualAccounts_TEST {
                 String FirstName1= 'ConFirstName ' + i;
                 String LastName = 'ConLastName ' + i;
 
-            	System.assertEquals(LastName + ' (' + FirstName1 + ') ' + ' Household', returnAccounts[i].Name);
+                System.assertEquals(LastName + ' (' + FirstName1 + ') ' + ' Household', returnAccounts[i].Name);
```
Comment (lifewithryan) [inline src/classes/ACCT_IndividualAccounts_TEST.cls]: @kdy618 same here with the order...

File: src/classes/ACCT_IndividualAccounts_TEST.cls (hunk 1 above)
Comment (kdy618) [reply src/classes/ACCT_IndividualAccounts_TEST.cls]: @lifewithryan , line 1748, FirstName1 is actually a variable so it's not really specifically calling/asking to return the 1st order of the firstname.

File: src/classes/ACCT_IndividualAccounts_TEST.cls (hunk 2 above)
Comment (kdy618) [reply src/classes/ACCT_IndividualAccounts_TEST.cls]: @lifewithryan , on line 1770, FirstName1 is actually not calling/asking to return the 1st order fo the first name, FirstName1 is a variable.

**Comment by spinkelman** [comment]
Passes branch QA - Will test further when merged to prove bug is fixed.
Closing WI and will test merged ACs with EDA regression

**Pull request merged**